crates/subspace-networking/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "subspace-networking"
-version = "0.1.0"
-authors = ["Nazar Mokrynskyi <nazar@mokrynskyi.com>"]
+version = "0.1.1"
```

Comment:
I think we normally won't bump the version at this stage and all crates will probably use `0.1.0` until the mainnet :P